docs/question/index.md

```diff
@@ -1,8 +1,14 @@
 # Questions
 
-**NOTE:** Any time you edit a question `info.json` file on a local copy of PrairieLearn, you need to click “Load from disk” to reload the changes. Edits to HTML or Python files can be picked up by reloading the page. You might need to generate a new variant of a question to run new Python code.
+Questions are the basic building blocks of PrairieLearn. They are the individual problems that students will solve, and they can be composed into assessments (homeworks and exams). Questions can be simple or complex, and they can be written in a variety of ways. This document describes how to create questions, including the directory structure, metadata, HTML templates, and server-side code.
 
-**NOTE:** New-style PrairieLearn questions are marked with `"type": "v3"`. This documentation only describes new-style questions, although old-style v2 questions are still supported in the code.
+## Question components
+
+A question is made up of three main components:
+
+1. **`info.json`**: Metadata about the question, including the question title, topic, and tags.
+2. **`question.html`**: The HTML template that defines the question. This is where you write the question text and define the input elements. Reference the [HTML documentation](html.md) for more details.
```

Comment:
Nit: would it make more sense to refer to it as the "template" documentation instead of the "HTML" documentation?

```suggestion
2. **`question.html`**: The HTML template that defines the question. This is where you write the question text and define the input elements. Reference the [template documentation](html.md) for more details.
```

Reply:
In 189fcab7d634a129d31bfb5c5fd1de4e2ee3cbd9 I renamed the file to `template.md`, and updated the links to say `question template documentation`.